### PR TITLE
Updated solutions to latest bits and VS2015

### PR DESCRIPTION
**Pull request by danderson00**

### CODE REVIEW

File: e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj [hunk 1]
```diff
@@ -43,13 +44,13 @@
     <AndroidLinkMode>SdkOnly</AndroidLinkMode>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.WindowsAzure.Mobile, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\WindowsAzure.MobileServices.2.0.0-beta\lib\monoandroid\Microsoft.WindowsAzure.Mobile.dll</HintPath>
+    <Reference Include="Microsoft.WindowsAzure.Mobile">
+      <HintPath>..\..\packages\WindowsAzure.MobileServices.2.0.0-beta-3\lib\monoandroid\Microsoft.WindowsAzure.Mobile.dll</HintPath>
```
Comment (phvannor) [inline e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj]: not your problem, but I keep asking on https://github.com/Azure/azure-mobile-apps-net-client/pull/8, is someone updating to the Apps nupkgs in this repo?

Comment (phvannor) [inline e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj]: well you did say latest bits too:)  So update this & Sql to the apps version & 2.0 GA?

Comment (fabiocav) [inline e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj]: +1

File: e2etest/test/Microsoft.WindowsAzure.MobileServices.WindowsStore.Test/Microsoft.WindowsAzure.Mobile.Win8.Test.csproj [hunk 2]
```diff
@@ -185,7 +185,7 @@
     <SDKReference Include="Microsoft.VCLibs, version=12.0">
       <Name>Microsoft Visual C++ 2013 Runtime Package for Windows</Name>
     </SDKReference>
-    <SDKReference Include="SQLite.WinRT81, Version=3.9.2">
```
Comment (fabiocav) [inline e2etest/test/Microsoft.WindowsAzure.MobileServices.WindowsStore.Test/Microsoft.WindowsAzure.Mobile.Win8.Test.csproj]: This should stay 3.9.2, unless there's another reason to downgrade.

File: e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj (hunk 1 above)
Comment (danderson00) [inline e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj]: huh... weird. this was the latest version that appeared in nuget.

**Comment by fabiocav** [comment]
Should this reference (resolve) #38?

**Comment by danderson00** [comment]
Yes. I opened that issue a couple of hours ago.

File: e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj (hunk 1 above)
Comment (phvannor) [inline e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj]: thats why we did the rename :) So services sites didn't get the GA apps sdks by accident.  You'll need to remove the service beta and add the apps GA

Comment (danderson00) [inline e2etest/test/Microsoft.WindowsAzure.MobileServices.Android.Test/Microsoft.WindowsAzure.Mobile.Android.Test.csproj]: lol oops

**Comment by fabiocav** [comment]
Did we miss the Xamarin projects with the last commit?

**Comment by danderson00** [comment]
Are you gents now happy with the PR?

**Comment by phvannor** [comment]
:+1: @pragnagopa you will need to rebase #41 as you and Dale are doing the same thing and will conflict with each other.

**Comment by fabiocav** [comment]
Looks good!

**Comment by danderson00** [comment]
Dang, sorry @pragnagopa. We need to reduce the chance of a conflict like this in future. Perhaps we should be tagging people on PRs who should be reviewing?

**Pull request merged**